config/eslintrc.json

```diff
@@ -63,18 +63,39 @@
         ],
         "import/extensions": ["off"],
         "no-undef": 1,
-        "no-unused-vars": 1
+        "no-unused-vars": "off"
```

Comment:
You did add or change quite a lot of settings here (like this one or above `camelcase` or `no-param-reassign`) where it is hard for me to understand why you changed them, but I assume they were necessary.

Reply:
Yes, they were all necessary. In the case of the `no-unused-vars` one, the TypeScript ESLint plugin contains a duplicate rule of its own. Some also interfered with TypeScript conventions. And a couple didn't seem to make sense and were only included because they were pulled in by the recommended or Airbnb plugin. For example, I had a class with some twelve or so properties. Requiring a line between all of them made it harder to group them, reducing readability.